pystac/single_file.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+
+from pystac.collection import Collection
+from pystac.io import STAC_IO
+from pystac.item import Item
+from pystac.item_collection import ItemCollection
+
+
+class SingleFile(ItemCollection):
```

Comment:
Perhaps this should this be rename to `SingleFileSTAC`? It would match the extension name better and be a bit more specific.

Reply:
done